novawallet/Modules/OperationDetails/OperationDetailsPresenter.swift

```diff
@@ -165,7 +165,24 @@ extension OperationDetailsPresenter: OperationDetailsPresenterProtocol {
     }
 
     func repeatOperation() {
```

Comment:
I think we can reuse send action. The idea is that we have only one handler for the main action that switch cases between models and calls appropriate handler